map_test.go

```diff
@@ -2316,37 +2327,101 @@ func ExampleMap_Iterate() {
 		entries = hash.Iterate()
 	)
 
+	m := make(map[string]uint32)
```

Comment:
Nit: we use `m` for `*Map` all over the place, I did a double take when I read this.
```suggestion
	values := make(map[string]uint32)
```